manim/mobject/types/vectorized_mobject.py

```diff
@@ -879,6 +880,49 @@ def get_subcurve(self, a, b):
         vmob.pointwise_become_partial(self, a, b)
         return vmob
 
+    def get_direction(self):
+        """
+        Uses shoelace in space_ops.py to calculate the direction.
+        The direction of points determines in which direction the
+        object is drawn, clockwise or counterclockwise.
+
+        Returns
+        -------
+        :class:`str`
+            Either `"CW"` or `"CCW"`.
+        """
+        return shoelace_direction(self.get_start_anchors())
+
+    def reverse_direction(self):
+        """
+        Reverts the point direction by inverting the point order.
+
+        Returns
+        -------
+        :class:`VMobject`
+            Returns self.
+        """
+        self.points = self.points[::-1]
+        return self
+
+    def force_direction(self, target_direction):
+        """
+        Makes sure that points are either directed clockwise or
+        counterclockwise.
+
+        Parameters
+        ----------
+        target_direction : :class:`str`
+            Either `"CW"` or `"CCW"`.
```

Comment:
```suggestion
            Either ``"CW"`` or ``"CCW"``.
```

Reply:
I've looked at the according generated part in the documentation and it looks fine. I also haven't seen an example of using 2 ` anywhere in our code or the guidelines in the wiki. Mind enlightening me first what the point of this change is?

Reply:
Double backticks are necessary to render as code in RST.

Reply:
Yes: in contrast to Github Markdown, code literals in ReST require blocks of double backticks. Blocks of single backticks are reserved for references.